### DISCUSSION TITLE
CI Chromium update very slow

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#11614] [Build / CI]
@mattpap is this normal? This was on a re-run (previous chromium update had failed entirely) 

<img width="665" alt="Screen Shot 2021-09-14 at 14 48 33" src="https://user-images.githubusercontent.com/1078448/133338548-e619db96-7359-4ec7-8b1f-cd5cae10a2b4.png">

**Comment by mattpap** [comment]
No it should take under a minute. Never have seen this being so slow before.

**Reply by bryevdv** [reply]
Maybe just a spurious issue today, I'll try to remember to keep an eye out to see if it continues
